src/components/color/Color.tsx

```diff
@@ -0,0 +1,33 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import * as _ from 'underscore';
+
+export interface ColorProps {
+    color: string;
+}
+
+// tslint:disable-next-line
+const VaporColors: {[key: string]: string} = require('!sass-variable-loader?preserveVariableNames!coveo-styleguide/scss/common/palette.scss');

```

Comment:
I think we can export this variable, we can use this one to select the color in other project after that